mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,13 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (default None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default True)
+        If True attempt to perform initial head localization (compute initial
+        evice to head coordinate transform) using HPI coils. If no
+        HPI coils are in info['dig'] hpi coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
> I think leaving it as head_loc makes sense. it is performing HPI localization and then setting info['dev_head_t], add_trans while technically correct isn't really meaningful.

But `head_loc` will not convey the meaning of what is going on. This could mean continuous head localization, etc. And even worse, `loc` is an entry in `info['chs'][ii]`.

How about `head_trans`, `add_head_trans`, `add_trans`, or `compute_trans`?

> If less then 3 chpi coils are on it doesn't issues a warning and sets the dev_head_t to identity.

In this case it should set it to `None` actually. It's possible (even if unlikely) to get identity in real data, `None` is used by MNE to mean "could not be computed" which is appropriate here.

> If you don't supply and digitization points and also get the ordering wrong (You plugged the coils into the wrong physical ports) then there is no way to catch that error. 

I thought in your code you had it do all permutations of coil orderings, so it should still work in this case, no?